_includes/android/getting-started.md

```diff
@@ -64,3 +64,11 @@ public class App extends Application {
   }
 }
 ```
+
+- **Option 3:** Setup `Application` as your application name in `AndroidManifest.xml`
+```xml
+<application ...
+  android:name=".App">
```

Comment:
I think this change can be integrated with `Option 1` on line 23, right?

Reply:
That would be fine